tests/e2e-playwright/tests/conftest.py

```diff
@@ -75,20 +76,27 @@ def pytest_addoption(parser: pytest.Parser) -> None:
         default=None,
         help="Service Key",
     )
+    group.addoption(
+        "--user-agent",
+        action="store",
+        type=str,
+        default="e2e-playwright",
```

Comment:
Thanks! Do you plan to modify the Grafana/Prometheus queries with this new value (rather than the puppeteer used before)?